conf/ocp_version/ocp-4.4-ga-config.yaml

```diff
@@ -7,7 +7,7 @@ RUN:
 DEPLOYMENT:
   ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
   installer_version: "4.4-ga"
-  ocp_channel: "stable"
+  ocp_channel: "fast"
```

Comment:
@Shrivaibavi could you please add more explanation of why this change?

Reply:
@RazTamir Added, Please check

Reply:
@RazTamir it's cause there is no promoted content to stable-4.4 channel. See mail thread:
http://post-office.corp.redhat.com/archives/aos-devel/2020-May/msg00071.html

@Shrivaibavi you can add comment pointing to this mail thread and the note that once there is build promoted to stable that we can revert back to stable.

Reply:
@petr-balogh Added comment session